js/bundle.go

```diff
@@ -46,8 +46,8 @@ type Bundle struct {
 // https://github.com/grafana/k6/issues/3065
 func (b *Bundle) checkMetricNamesForPrometheusCompatibility() {
 	const (
-		nameRegexString = "^[a-zA-Z_][a-zA-Z0-9_]{1,63}$"
-		badNameWarning  = "Metric name should only include ASCII letters, numbers and underscores. " +
+		nameRegexString = "^[a-zA-Z_][a-zA-Z0-9_]{1,128}$"
```

Comment:
```suggestion
		nameRegexString = "^[a-zA-Z_][a-zA-Z0-9_]{1,255}$"
```
Is there a specific reason behind it? If not then I would prefer if we could keep it aligned with OTel.

Reply:
Yes that is the current limit in the registry. The place where this limit will go after this becomes an error instead of a warning.

Reply:
My suggestion was about the size limit `s/128/255/`

Reply:
Yes this is hte current limit - 128

https://github.com/grafana/k6/blob/981b770946f5df5d772554e5906f888ef1ca8a94/metrics/registry.go#L29

Reply:
Sorry, but I'm not still getting why we are not just bumping everything to 255. What is the reason to keep as 128? If there is one technical then I would like us adding a comment on the registry's regex explaining it. Prometheus doesn't have any limit and the OpenTelemetry limit is set to 255 so it sounds to me we should just match the OTel limit.